editions/tw5.com/tiddlers/macros/UnusedTitleMacro.tid

```diff
@@ -10,7 +11,29 @@ It uses the same method as the create new tiddler button, a number is appended t
 
 !! Parameters
 
-;baseName
-: A string specifying the desired base name, defaulting to `New Tiddler`
+; baseName
+: A string specifying the desired base name, defaulting to `New Tiddler`. <br>The default setting can be adjusted in the $:/ControlPanel '': Info : Basics - tab.''
+
+; separator
+: <<.from-version "5.1.24">> An ''optional'' string specifying the separator between baseName and the unique number. eg: `separator:"-"`. Defaults to a space: `" "`. If you need an empty separator use the ''template''!
+
+; template
+: <<.from-version "5.1.24">> A ''optional'' template string can be used to allow you maximum flexibility. If the template string is used, there will always be a counter value. 
+
+!! Template String
+
+; `$baseName$`
```

Comment:
I understand why you've gone with mixed case here, but I think it would be better to be consistent with the other places that we do `$x$` substitution, and use lowercase consistently. It wouldn't be unreasonable to use case-insensitive matching so that if people were to understandably write `$baseName$` then things would still work.